schemas/context/pushdetail.schema.json

```diff
@@ -0,0 +1,54 @@
+{
+    "meta:license": [
+        "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+        "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+        "you may not use this file except in compliance with the License. You may obtain a copy",
+        "of the License at https://creativecommons.org/licenses/by/4.0/"
+    ],
+    "$id": "https://ns.adobe.com/xdm/context/pushdetail",
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "title": "Push notification detail",
+    "type": "object",
+    "meta:extensible": true,
+    "description": "Push notification detail required to send mobile push notification to a specific mobile application on a given device.",
+    "definitions": {
+        "pushdetail": {
+            "properties": {
+                "xdm:pushToken": {
+                    "title": "Push Token",
+                    "type": "string",
+                    "description": "Valid push token for the given push platform."
+                },
+                "xdm:pushPlatform": {
+                    "title": "Push Platform",
+                    "type": "string",
+                    "description": "Push provider, e.g. apns or fcm"
+                },
+                "xdm:isBlacklisted": {
+                    "title": "Blacklist Flag",
+                    "type": "boolean",
+                    "description": "Indicates whether the push token is blacklisted."
+                },
+                "xdm:appId": {
+                    "title": "Application Identifier",
+                    "type": "string",
+                    "description": "Unique Identifier for the mobile application."
+                },
+                "xdm:id": {
```

Comment:
Is this the Notification Id?

Reply:
Realistically this would be the ECID that was associated with this push token, so we can still know the ecid for the push token after profile merging was done.

Reply:
Then call it ECID - you *cannot* use `xmd:id` it's _reserved_